securedrop/sass/_base.sass

```diff
@@ -2,7 +2,6 @@
 
 // LIBRARIES
 @import libraries/normalize
-@import libraries/font-awesome-iconography
```

Comment:
ah so what we want to do here is move this `font-awesome` import into `journalist.sass`. the reason being is that you are correct we do not use anything in the `font-awesome` library in the source interface but: we _do_ on the journalist interface (127.0.0.1:8081 in the dev env), e.g. these icons are all font awesome icons:

<img width="595" alt="Screen Shot 2019-05-25 at 11 42 47 AM" src="https://user-images.githubusercontent.com/7832803/58373426-064e9500-7ee3-11e9-9b84-dc94ff02a8ff.png">